js/compileShell.py

```diff
@@ -578,6 +578,7 @@ def fixateVer(pcfile):
         """Returns the current version number (47.0a2)."""
         with open(pcfile, 'rb') as f:
             for line in f:
+                line = line.decode("utf-8", "replace")
```

Comment:
I kept the 'rb' mode for [Windows compatibility](https://docs.python.org/2/tutorial/inputoutput.html#reading-and-writing-files).